app/views/culinary_recipes/show.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="mx-auto md:w-2/3 w-full flex">
+
+<%= form_with model: @culinary_recipe, url: toggle_public_status_culinary_recipe_path(@culinary_recipe), method: :patch, local: false do |f| %>
+  <%= f.check_box :public, { onchange: "this.form.submit();" } %>
+  <%= f.label :public, 'Public' %>
+  <%= f.submit "Submit", class: "hide-submit-button"%>
+<% end %>
+
+  <div class="mx-auto w-full">
+
+    <%= render @culinary_recipe %>
+
+    <%= link_to "Back to culinary_recipes", culinary_recipes_path, class: "ml-2 mt-10 rounded-lg py-3 px-5 bg-gray-100 inline-block font-medium" %>
+
+    <%= link_to 'Generate Shopping List', method: :post, class: "l-2 mt-10 rounded-lg py-3 px-5 bg-gray-100 inline-block font-medium" %>
```

Comment:
- [ ] The progress on the job is good so far, but some changes are still needed. According to the project requirement, when the user clicks the `Generate Shopping List` button, they should be taken to a page that displays a list of food items that are missing for all of the recipes of the currently logged-in user. Additionally, the page should show the total number of food items that are missing and the total cost of these items. Please update your code to reflect these requirements and create a page that matches the wireframe.

https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/general_shopping_list.png